### PR TITLE
Code Improvements for Pending Source Management

### PR DESCRIPTION
**Pull request by nathandyer**
## Status

Work in progress

## Description of Changes

This is a bite-size PR based on additional changes requested in PR #6785

## Testing

- [ ] CI passes

From my perspective, I don't believe additional testing beyond making sure the tests pass are needed, as this does not make any substantial change to the logic or functionality.

### If you made changes to the server application code:
- [x] Linting (`make lint`) and tests (`make test`) pass in the development container

### CODE REVIEW

File: securedrop/management/sources.py [hunk 1]
```diff
@@ -10,20 +10,17 @@
 def remove_pending_sources(args: argparse.Namespace) -> int:
     """
     Removes pending source accounts, with the option of keeping
-    the n newest source accounts.
+    the n newest source accounts (as provided by args.keep_most_recent).
```
Comment (cfm) [inline securedrop/management/sources.py]: Suggest:

```suggestion
    the `args.keep_most_recent` newest source accounts.
```

**Review by cfm — DISMISSED** [review]
I'll defer to @legoktm to review the changes he requested in <https://github.com/freedomofpress/securedrop/pull/6785#pullrequestreview-1419188856>.  But I thought I'd note, @nathandyer, that this branch is currently failing `lint`.

**Comment by nathandyer** [comment]
Sorry about that @cfm, apparently it wanted one extra blank line :upside_down_face: 

Linter should now be happy, and this should be ready for review whenever @legoktm has a chance. Thanks both!

**Review by zenmonkeykstop — APPROVED** [review]
Changes look good based on visual review, thanks for squishing :)

**Comment by legoktm** [comment]
Belated 👍🏾 from me :)

**Pull request merged**